From 6efa5b120449974b46e05113cab249f1b0b11da6 Mon Sep 17 00:00:00 2001
From: Kevin Chen <kevin_chen@aspeedtech.com>
Date: Fri, 22 Aug 2025 10:02:41 +0800
Subject: [PATCH 13/32] misc: edaf_bridge: Fix typo

Corrects a typo in the 'edaf-erase-with-1' property name, replacing
'misspelled 'edaf-earse-with-1' in both the device tree source and
the associated driver code.

Change-Id: I9111bd0c3e59fc7a3236be502f2214edea7077b5
Signed-off-by: Kevin Chen <kevin_chen@aspeedtech.com>
(cherry picked from commit 6b9b9585cf521dc02ea354c5339d88fcdfe71930)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/riscv/dts/ast2700-ibex.dts   |  2 +-
 drivers/misc/aspeed_edaf_bridge.c | 10 +++++-----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/riscv/dts/ast2700-ibex.dts b/arch/riscv/dts/ast2700-ibex.dts
index 7cd3520723..e777a5fa48 100644
--- a/arch/riscv/dts/ast2700-ibex.dts
+++ b/arch/riscv/dts/ast2700-ibex.dts
@@ -241,7 +241,7 @@
 	espi-base = <0x0 0x14c05000>;
 #if 0 //DDR mode using edaf0 reserved memory
 	edaf-ddr-mode;
-	edaf-earse-with-1;
+	edaf-erase-with-1;
 	mem-base = <&edaf0>;
 #else //Flash mode using SPI0
 	mem-base = <0x1 0x80000000>; //SPI0
diff --git a/drivers/misc/aspeed_edaf_bridge.c b/drivers/misc/aspeed_edaf_bridge.c
index 7fa02b84ad..061856da2f 100644
--- a/drivers/misc/aspeed_edaf_bridge.c
+++ b/drivers/misc/aspeed_edaf_bridge.c
@@ -41,8 +41,8 @@
 #define   EDAF_BDGE_MISC_CBASE_H	GENMASK(7, 0)
 
 #define EDAF_BDGE_GLOBAL_CFG		0x0
-#define   EDAF_BDGE_GLOBAL_CFG0_EARSE	BIT(1)
-#define   EDAF_BDGE_GLOBAL_CFG1_EARSE	BIT(9)
+#define   EDAF_BDGE_GLOBAL_CFG0_ERASE	BIT(1)
+#define   EDAF_BDGE_GLOBAL_CFG1_ERASE	BIT(9)
 
 uint64_t ast27xx_soc_virt_addr_to_phy_addr(uintptr_t addr)
 {
@@ -158,7 +158,7 @@ static int aspeed_edaf_bridge_probe(struct udevice *dev)
 			return -ENOMEM;
 		}
 
-		if (ofnode_read_bool(node, "edaf-earse-with-1")) {
+		if (ofnode_read_bool(node, "edaf-erase-with-1")) {
 			rc = ofnode_read_u32(node, "aspeed,edaf_bridge_cfg", &phandle);
 			if (rc) {
 				printf("cannot get edaf_bridge_cfg phandle\n");
@@ -179,9 +179,9 @@ static int aspeed_edaf_bridge_probe(struct udevice *dev)
 
 			edaf_gcfg = readl(edaf_bridge_cfg_regs + EDAF_BDGE_GLOBAL_CFG);
 			if (plat == EDAF_BDGE_OVER_ESPI0) {
-				edaf_gcfg |= EDAF_BDGE_GLOBAL_CFG0_EARSE;
+				edaf_gcfg |= EDAF_BDGE_GLOBAL_CFG0_ERASE;
 			} else if (plat == EDAF_BDGE_OVER_ESPI1) {
-				edaf_gcfg |= EDAF_BDGE_GLOBAL_CFG1_EARSE;
+				edaf_gcfg |= EDAF_BDGE_GLOBAL_CFG1_ERASE;
 			} else {
 				printf("Unknown platform for eDAF bridge\n");
 				return -ENODEV;
-- 
2.34.1

